### PR TITLE
Login screen changes

### PR DESCRIPTION
**Pull request by marusak**
Honestly, I am not sure about either of those commits, so lets see.
1. Use standard background on RHEL - For "before" see #13872, now
rhebackground
I am not sure why there was special background for rhel. Is there some good reasoning behind this? Was this special case back when cockpit logo was used as a background? Should it be still just black? @martinpitt maybe knows
2. login: Drop the dark rectangle - I always thought that the image just did not start right from the side of the screen. But it seems there was this darker rectangle for some reason. It seems suspiciously purposeful, thus no idea why it was there. @garrett might know.
Before:
rhebackground
After:
withoutblackline

### CODE REVIEW

File: src/branding/rhel/Makefile.am [hunk 1]
```diff
@@ -2,7 +2,6 @@ rhelbrandingdir = $(datadir)/cockpit/branding/rhel
 
 rhelbranding_DATA = \
 	src/branding/rhel/branding.css \
-	src/branding/rhel/bg-login.jpg \
```
Comment (martinpitt) [inline src/branding/rhel/Makefile.am:5]: :+1 to that -- this really looks like an obsolete placeholder -- what's the point in shipping a jpg that is entirely black..

**Review by martinpitt — COMMENTED** [review]
I'm good with both of these changes, thanks! Let's wait for @garrett's confirmation as welll, though. Please add a Closes # though, or send these as two PRs.

**Comment by garrett** [comment]
> But it seems there was this darker rectangle for some reason.

Because it was there in PF4 when I implemented the login screen.

It's not there now, so either they changed their mind on it, or it was a bug (that I copied).

**Comment by garrett** [comment]
I noticed that we also have a regression:

Screenshot_2020-09-07 rain

That white space shouldn't be there. (This is in master.)

I'm building the branch now in a VM test it, but it'll take a while on my laptop.

**Comment by marusak** [comment]
> It's not there now, so either they changed their mind on it, or it was a bug (that I copied).

Are you fine with these changes then?

> That white space shouldn't be there. (This is in master.)

Right, I saw this one before as well. But unrelated to these changes, lets fix it in another PR.

**Comment by garrett** [comment]
> Are you fine with these changes then?

I think so? 

As I said, I'm building this branch. It will take a long while before I can view these changes.

 I'm fine with it in principle, as the darker border was copied from PF4 (whether it was intentional there or not) and is now no longer in PF4, so removing it makes sense to re-sync. And I don't know why RHEL had a different (black) background. Perhaps that's from the PF3 days?

> lets fix it in another PR

OK.

**Review by garrett — APPROVED** [review]
Built a RHEL 8.3 testing VM; it looks correct here, with the same background as the rest and also without the shaded left side.

:+1:

**Pull request merged**